frontend/src/pages/InboxPage.jsx

```diff
@@ -69,18 +67,16 @@ const InboxPage = () => {
         )
     }
 
-    const color = getPaletteColor(theme.type, data?.color)
+    const color = theme.grey
 
     return (
         <>
             <TitleBox>
                 <PageTitleBox>
-                    <PageTitle $color={color}>{data.name}</PageTitle>
-                    <PrivacyIcon
-                        privacy={data.privacy}
-                        color={color}
-                        isProject
-                    />
+                    <PageTitle $color={color}>{t("inbox")}</PageTitle>
+                    <ProjectPrivacyBox $color={color}>
+                        <FeatherIcon icon="lock" />
+                    </ProjectPrivacyBox>
```

Comment:
`PrivacyIcon`을 그대로 사용하는 게 더 좋을 것 같습니다.

```suggestion
                    <PrivacyIcon
                        privacy="private"
                        color={color}
                        isProject
                    />
```